app/bt/AffectedUserFlow/CodeInput.spec.tsx

```diff
@@ -18,4 +24,77 @@ describe('CodeInputScreen', () => {
     expect(getByTestId('affected-user-code-input-screen')).not.toBeNull();
     expect(getByTestId('code-input')).toHaveTextContent('');
   });
+
+  describe('validates the verification code', () => {
+    it('informs of an invalid code error', async () => {
+      const error = 'InvalidCode' as const;
+      const wrongTokenResponse = {
+        kind: 'failure' as const,
+        error,
+      };
+      jest
+        .spyOn(API, 'postVerificationCode')
+        .mockResolvedValueOnce(wrongTokenResponse);
+
+      const { getByTestId, getByLabelText, getByText } = render(
+        <AffectedUserProvider>
+          <CodeInputScreen />
+        </AffectedUserProvider>,
+      );
+      fireEvent.changeText(getByTestId('code-input'), '12345678');
+      fireEvent.press(getByLabelText('Submit'));
```

Comment:
Amanda and I are writing tests for the onboarding flow. Since most of the text is from translations, is there a good way to use the translations, or is hardcoding a fine approach?

Reply:
Not having translations in specs makes them more readable and the intent of the specs isnt to test translations per say, so I prefer having the actual text.  And simply updating the specs if the copy updates.

Reply:
Good question, this crossed my mind but transforming things and exploding builds side-tracked me 😂 In the past I usually mock the `t` function and search for the keys, but that puts us in a pickle where we might not have the key added in at least the default translation so I would vote for either having it this way or for mocking the function and validating that translations keys are present but I don't have an idea on how to do that. Happy to brainstorm some though 😄

Reply:
I am down with this way - wanted some validation. This is great!